### PR TITLE
Doc: Clarify some installation instructions

### PR DESCRIPTION
**Pull request by CommonLoon102**
Make a note about the `C++ CMake tools for Windows` requirement, and fix the path for the SDL libs.

### CODE REVIEW

File: doc/building_windows.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ You have two main options in how you want to build Julius. You can either:
 
 1. Make sure you have [Microsoft Visual Studio](https://visualstudio.microsoft.com/vs/) installed.
    Please note that only Visual Studio 2017 (fully updated) or Visual Studio 2019 can be directly used.
+   You need to have `C++ CMake tools for Windows` installed for Visual Studio what you can install via
```
Comment (crudelios) [inline doc/building_windows.md]:
```suggestion
   You need to have `C++ CMake tools for Windows` installed for Visual Studio that you can install via
```

Fix typo.

**Comment by crudelios** [comment]
Hey, thanks for the suggestion. I may be wrong, but are the `C++ CMake tools for Windows` a requirement also on MSVC 2019?

I don't remember installing them (it's been a while since I installed it though) and it's always worked for me.

**Comment by crudelios** [comment]
Hey, thanks for the suggestion. I may be wrong, but are the `C++ CMake tools for Windows` a requirement also on MSVC 2019?

I don't remember installing them (it's been a while since I installed it though) and it's always worked for me.

**Comment by CommonLoon102** [comment]
> 
> 
> Hey, thanks for the suggestion. I may be wrong, but are the `C++ CMake tools for Windows` a requirement also on MSVC 2019?
> 
> I don't remember installing them (it's been a while since I installed it though) and it's always worked for me.

Yes, I used MSVC 2019 and I had to install it. Maybe it was originally checked by default when I installed VS the first time and I unchecked it at that time, I don't remember.

**Pull request merged**